posthog/clickhouse/test/__snapshots__/test_schema.ambr

```diff
@@ -108,8 +108,7 @@
       
       
   ) ENGINE = Kafka('test.kafka.broker:9092', 'clickhouse_events_json_test', 'group1', 'JSONEachRow')
-  
-      SETTINGS kafka_skip_broken_messages = 100
```

Comment:
What happens now we drop this setting? Can invalid messages choke up the main MV?

Reply:
This setting is at odds with streaming errors. Effectively we'll no longer crash from errors, they'll all go to the DLQs, for which we should have alerting